src/thriveopedia/pages/wiki/OrganelleLinkButton.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using Godot;
+
+/// <summary>
+///   Button which links to an organelle Thriveopedia page.
+/// </summary>
+public class OrganelleLinkButton : VBoxContainer
+{
+    [Export]
+    public NodePath? ButtonPath;
+
+    [Export]
+    public NodePath LabelPath = null!;
+
+#pragma warning disable CA2213
+    private Button button = null!;
+    private Label label = null!;
+#pragma warning restore CA2213
+
+    public OrganelleDefinition Organelle { get; set; } = null!;
```

Comment:
I see this quite a bit in the changes here, that there's a public property with a setter that is null overridden. I can't immediately remember that similar things would have been necessary for other GUI parts. In terms of code encapsulation this is kind of ringing alarm bells in my head, but I can't immediately remember how this kind of design is handled elsewhere (as to what could potentially be done).

Reply:
I wondered that too, but it's a definitely a pattern that's present elsewhere, e.g. Asset in GalleryCard.

Reply:
I guess it's fine then if neither of us has a better idea how to do this and this is an approach that's already used.